cfme/tests/configure/test_database_ui.py

```diff
@@ -23,8 +23,8 @@ def test_configure_vmdb_last_start_time(appliance):
                 "journalctl -u {}-postgresql.service  --boot=0 | sed '4!d'".format(item))
 
     ui_last_start_time = parser.parse(view.summary('Properties').get_text_of('Last Start Time'))
-
-    tz = pytz.timezone(appliance.ssh_client.run_command("date +'%Z'").output.strip())
+    tz = pytz.timezone(appliance.ssh_client.run_command("timedatectl | grep 'Time zone'")
```

Comment:
**QUESTION:** What was wrong with the original method?   I would have thought that would have been correct (obviously it wasn't because you are changing it...just trying to understand the exact nature of the error).   This might be a place to add a comment about why use timedatectl above other utilities.

Reply:
yup original method worked well in prt run, but after jenkins run we got UnknownTimeZoneError, as it turned out timezone name that we got not worked well for object creation in some cases, with the help of timedatectl we can get full timezone name that much better and safely can be used here

Reply:
*REQUIRED:* Please add a comment describing what you just described.

Reply:
Added a comment here